pkg/apis/componentconfig/types.go

```diff
@@ -554,3 +554,71 @@ const (
 	// "kube-scheduler" is the default scheduler lock object name
 	SchedulerDefaultLockObjectName = "kube-scheduler"
 )
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// EncryptionConfiguration stores the complete configuration for encryption providers.
+type EncryptionConfiguration struct {
```

Comment:
this name is far too generic for a global API group.

Reply:
> this name is far too generic for a global API group.

What did you mean by "a global API group"? It's not global as far I understand.
Do you have a better name instead?